### PR TITLE
modulesync 4.1.0

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

**Review by llowder — APPROVED** [review]

**Comment by bastelfreak** [comment]
depends on https://github.com/voxpupuli/puppet-nomad/pull/50

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -8,8 +8,6 @@
 
 require 'voxpupuli/test/spec_helper'
 
-add_mocked_facts!
```
Comment (ekohl) [inline spec/spec_helper.rb:11]: This was added manually. It may break things.

Comment (bastelfreak) [reply spec/spec_helper.rb:11]: the unit tests still pass, so that looks fine.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -58,10 +62,6 @@ jobs:
         puppet: ${{fromJson(needs.setup_matrix.outputs.puppet_major_versions)}}
     name: ${{ matrix.puppet.name }} - ${{ matrix.setfile.name }}
     steps:
-      - name: Enable IPv6 on docker
-        run: |
-          echo '{"ipv6":true,"fixed-cidr-v6":"2001:db8:1::/64"}' | sudo tee /etc/docker/daemon.json
-          sudo service docker restart
```
Comment (genebean) [inline .github/workflows/ci.yml:64]: Do you really want this to go away?

Comment (ekohl) [reply .github/workflows/ci.yml:64]: It was removed because in some cases it lead to problems.

Comment (bastelfreak) [reply .github/workflows/ci.yml:64]: We used this prefix in the past so stuff has at least one address. Without that ipv6 was completely disabled on travis I think and we copied it to github actions. But linux routing stack is a jerk and if we add this and the container want's to speak to the outside world, and the outside world has an AAAA DNS record, the docker container tries IPv6 and waits ages until it runs in a timeout. In a perfect world, cloud providers would route ipv6 prefixes to the link local address of a hypervisor and we would get a global unicast address in the containers. Sadly IPv6 support on azure/gcp/aws is meh :crying_cat_face:

**Review by attachmentgenie — APPROVED** [review]

**Pull request merged**